pkgs/development/libraries/libbladeRF/default.nix

```diff
@@ -21,6 +21,11 @@ stdenv.mkDerivation rec {
   # Fixup shebang
   prePatch = "patchShebangs host/utilities/bladeRF-cli/src/cmd/doc/generate.bash";
 
+  # Fixes macos and freebsd compilation issue.
+  # This is fixed upstream but hasn't been released yet.
+  # https://github.com/Nuand/bladeRF/commit/0cb4ea888543b2dc75b876f7024e180854fbe9c3
+  patches = ./fix-OSX-and-FreeBSD-compilation-issue.patch;
```

Comment:
It is better to use `fetchPatch` here instead of providing the file.

Reply:
Makes sense. Fixed.